frontend/pages/admin/TeamManagementPage/TeamDetailsWrapper/UsersPage/UsersPage.tsx

```diff
@@ -76,36 +77,36 @@ const MembersPage = ({ location, router }: ITeamSubnavProps): JSX.Element => {
   const [editUserErrors, setEditUserErrors] = useState<IUserFormErrors>(
     DEFAULT_CREATE_USER_ERRORS
   );
-  const [memberIds, setMemberIds] = useState<number[]>([]);
+  const [userIds, setUserIds] = useState<number[]>([]);
 
   const toggleAddUserModal = useCallback(() => {
-    setShowAddMemberModal(!showAddMemberModal);
-  }, [showAddMemberModal, setShowAddMemberModal]);
+    setShowAddUserModal(!showAddUserModal);
+  }, [showAddUserModal, setShowAddUserModal]);
 
-  const toggleRemoveMemberModal = useCallback(
+  const toggleRemoveUserModal = useCallback(
     (user?: IUser) => {
-      setShowRemoveMemberModal(!showRemoveMemberModal);
+      setShowRemoveUserModal(!showRemoveUserModal);
       user ? setUserEditing(user) : setUserEditing(undefined);
     },
-    [showRemoveMemberModal, setShowRemoveMemberModal, setUserEditing]
+    [showRemoveUserModal, setShowRemoveUserModal, setUserEditing]
   );
 
   // API CALLS
 
   const {
-    data: members,
-    isLoading: isLoadingMembers,
-    error: loadingMembersError,
+    data: users,
```

Comment:
How about `teamUsers`, etc. on this page, to disambiguate from other users api calls?

Reply:
sgtm